common/fs/src/tail.rs

```diff
@@ -106,49 +106,124 @@ impl Tailer {
+                                            Lookback::SmallFiles => {
+                                                let offset = match initial_offsets.as_ref() {
+                                                    Some(initial_offsets) => {
+                                                        let offset = initial_offsets.get(&path.as_os_str().as_bytes().into()).copied().unwrap_or(0);
+                                                        debug!("Got offset {} from state for {:?} using key {:?}", offset, name, path);
+                                                        offset
                                                     }
+                                                    None => {
+                                                        let len = path.metadata().map(|m| m.len()).unwrap_or(0);
+                                                        debug!("Smallfiles lookback {} from state for {:?} using key {:?}", len, name, path);
+                                                        if len < 8192 {
+                                                            0
+                                                        } else{
+                                                            len
+                                                        }
+                                                    }
+                                                };
+                                                info!("initialized {:?} with offset {}", path, offset);
+                                                data.borrow_mut().deref_mut().seek(offset).await.unwrap_or_else(|e| error!("error seeking {:?}", e))
+                                            },
+                                            Lookback::None => {
+                                                let len = path.metadata().map(|m| m.len()).unwrap_or(0);
+                                                info!("initialized {:?} with offset {}", path, len);
+                                                data.borrow_mut().deref_mut().seek(len).await.unwrap_or_else(|e| error!("error seeking {:?}", e))
+                                            }
+                                        }
+                                        if fs.is_initial_dir_target(&path) {
+                                            debug!("File is in initial dirs");
+                                            data.borrow_mut().tail(vec![path]).await
+                                        } else {
+                                            None
+                                        }
+                                    }
+                                    Entry::Symlink { name, link, .. } => {
+                                        let sym_path = path.clone();
+                                        let sym_name = name;
+                                        info!("Initialise event for symlink {:?}, target {:?}", name, link);
```

Comment:
We should unify `Initialise` vs `initialize` :) 

Messages in logs show and mixed casing, for example:

```
[2021-03-26T10:41:09Z INFO  fs::tail] initialized "/var/lib/docker/containers/f527c4bac8f3d84606a21d08016227f989a7fe61f19107c9324ff1ea4426dba6/f527c4bac8f3d84606a21d08016227f989a7fe61f19107c9324ff1ea4426dba6-json.log" with offset 34274
[2021-03-26T10:41:09Z INFO  fs::tail] Initialise event for symlink "7.log", target "/var/lib/docker/containers/f527c4bac8f3d84606a21d08016227f989a7fe61f19107c9324ff1ea4426dba6/f527c4bac8f3d84606a21d08016227f989a7fe61f19107c9324ff1ea4426dba6-json.log"
```